### PR TITLE
add files for fetch_tagged_entity_dataset pipeline.

### PR DESCRIPTION
**Pull request by greed2411**
ignore the branch name, but adding a new pipeline called `fetch_tagged_entity_dataset`. it gives modified entity tog dataset which is ready for eval (for changelog.md)

This takes a tog job, and the annotated region tagged texts & entity types -> hits duckling to get the `value` parameter.
it also mutates the predicted entities structure to be sane and comparable with truth.

the resulting columns which are ready for evaluation: `truth_entities_with_duckling` & `predicted_entities_with_modifications` can be compared directly using eevee.

successful runs for this pipeline (for american finance & mahindra finance) are here:
https://kubeflow.skit.ai/pipeline/#/runs/details/74db9af8-2963-47ec-8c94-c6b85b2cc002
https://kubeflow.skit.ai/pipeline/#/runs/details/491ebe4a-53bf-486e-8317-c3ba9ccd64a0

### CODE REVIEW

**Comment by greed2411** [comment]
cc: @nilesh-skit @piyushmakhija5 @janaab11

File: skit_pipelines/components/modify_tog_entities.py [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+        return df
+        
+    df = pd.read_csv(data_path)
+
+    logger.info(f"duckling running at: {pipeline_constants.DUCKLING_HOST}")
+
+    mod_df = modify_truth(df)
+    mod_df = modify_predictions(mod_df)
+    mod_df.to_csv(output_path, index=False)
+
```
Comment (biswaroop1547) [inline skit_pipelines/components/modify_tog_entities.py:248]: this component file looks too big, can you break it down to different modules and keep it inside a directory? something similar to the `tag_calls` component.

**Review by biswaroop1547 — CHANGES_REQUESTED** [review]

File: skit_pipelines/components/modify_tog_entities.py (hunk 1 above)
Comment (greed2411) [reply skit_pipelines/components/modify_tog_entities.py:31]: done.

**Comment by greed2411** [comment]
have made the changes requested, please lmk if there is anything else.

**Review by biswaroop1547 — APPROVED** [review]

**Pull request merged**